xarray/backends/pynio_.py

```diff
@@ -28,6 +29,17 @@ def get_array(self):
         return self.datastore.ds.variables[self.variable_name]
 
     def __getitem__(self, key):
+        if isinstance(key, indexing.VectorizedIndexer):
```

Comment:
@shoyer 
Thanks for your suggestion. Now it raises a NotImplementedError.
This backend seems to partly support outer indexing.